src/tasks/wifi.c

```diff
@@ -437,7 +437,7 @@ static void send_beacon(struct Serial* serial, const char* ips[])
                 for(++ips; *ips && 0 == **ips; ++ips);
                 const bool more = !!*ips;
                 /* Don't add empty strings to the array */
-                if (0 != *ip)
+                if (0 != *ip && !STR_EQ(ip, IPV4_NO_IP_STR))
```

Comment:
Yes we have been doing that for some time now and it is just something that we shouldn't do.  So I went ahead and fixed it.